compiler/runtime/OMRRuntimeAssumptions.cpp

```diff
@@ -28,7 +28,7 @@
 #define ASM_CALL
 #endif
 
-#if defined(TR_HOST_S390) || defined(TR_HOST_X86) || defined(TR_HOST_ARM)
+#if defined(TR_HOST_S390) || defined(TR_HOST_X86) || defined(TR_HOST_ARM) || defined(TR_HOST_ARM64)
```

Comment:
Could you update the copyright header on this file to 2019 please?

Reply:
Sorry, I forgot to update this. Updated the copyright year.